code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -44,3 +44,263 @@
 /mob/living/simple_animal/hostile/retaliate/clown/attack_hand(mob/living/carbon/human/M)
 	..()
 	playsound(src.loc, 'sound/items/bikehorn.ogg', 50, 1)
+
+/mob/living/simple_animal/hostile/retaliate/clown/lube
+	name = "Living Lube"
+	desc = "A puddle of lube brought to life by the honkmother."
+	icon = 'icons/mob/clown_mobs.dmi'

```

Comment:
move the original clown sprite to this file, set the original clown mob's `icon` to this file, then remove this line from every subtype

Reply:
(not required but will save lines)